code/__DEFINES/flags.dm

```diff
@@ -137,8 +137,11 @@
 #define FLAMMABLE 0
 #define ON_FIRE 1
 
-//resistance_flags
-#define INDESTRUCTIBLE 64 //doesn't take damage
+//Fire and Acid stuff, for resistance_flags
+#define UNACIDABLE		(1<<4) //acid can't even appear on it, let alone melt it.
```

Comment:
Note to future coders; this format is preferable to the previously 1, 2, 4, 8, 16, 32.

Reply:
Sorry for the bother but would you mind elaborating on that a bit? Is it for readability purposes or does it present more usable flags than the 1 2 4 8 16 32 etc. format?

Just curious! I've seen it in Baycode but don't understand

Reply:
It's for readability, mostly.

It tells us just how many bits we've used up, and is a lot easier when you get into higher numbers.

Reply:
Thanks, I appreciate the explanation